Topic Wise Questions with Solutions/Array/reverse_array.py

```diff
@@ -10,13 +10,10 @@
 
 
 def reverseList(A, start, end):
-	while start < end:
-		A[start], A[end] = A[end], A[start]
-		start += 1
-		end -= 1
+	return A.reverse()
 
 # Driver function to test above function
-A = list(map(int,input().split("")))
+A = list(map(int,input().split(" ")))
 print(A)
 reverseList(A, 0, len(A))
 print("Reversed list is")

```

Comment:
`
#In this code we will be getting input as a string
input_string = input("Enter the string to be reversed")# getting the input
output_string = input_string[::-1]#Reversing the string using slicing operation
print(output_string)#prints the output

#Sample Case:
#input_string = aeiou
#output_string = uoiea

# In the below code ,we will have mutiple input and we will be reversing them and printing them as a string.
input_string1 = list(map(str,input("Enter the elements of the string with spaces.").split()))#getting multiple input
output_string1 = input_string1[::-1]#reversing the list using slice operation
#we ca also do reversing of list with in-built functions ( i.e list1 = [1,2,3,4] list1.reverse() reverses the list )
print(" ".join(output_string1))#joins the reversed list to an empty string and prints as output

#Sample Case:p
#input_string1 = 1 2 3 4 5
# it is stored as input_string1 = [1,2,3,4,5]
# output stored as output_string1 = [5,4,3,2,1]
#output is printed as 54321

`